README.md

```diff
@@ -1,10 +1,18 @@
 
 # find-your-national-insurance-number-frontend
 
-This is a placeholder README.md for a new repository
+## About
+This is the frontend repository for the Find your National Insurance number service. It guides users to find their National Insurance numbers online where possible, and via a letter where not. 
 
-### License
+## How to run locally
+To run locally, start this service and dependant services through the service manager with the profile `FMN_ALL`.
```

Comment:
Can we please add the profile for this service? 

I.e. If we want to stop it and debug we want tot know which service to stop and which port this front end runs on?